k8s.gcr.io/README.md

```diff
@@ -79,7 +99,7 @@ To promote an image, follow these steps:
 1. Merge the PR. Your image will be promoted by one of two jobs:
    - [`post-k8sio-image-promo`][post-promo-job] is a postsubmit that runs immediately after merge
    - [`ci-k8sio-cip`][ci-promo-job] is a postsubmit that runs immediately after merge
-1. A periodic 
+1. A periodic
```

Comment:
I don't understand this fragment?

Reply:
It was like this when I got here!  Looks like fixing trailing whitespace makes it my responsibility to reword.

Added a commit to redo this slightly